### DISCUSSION TITLE
crazyflie example

### DISCUSSION THREAD

**Discussion by 4perros** [bitcraze/discussions#885] [General]
hello Marcus
I found this website with an interesting example for the Crazyflie i want to know if it is safe to flash on the aideck and on the crazyflie. The link for it is:
AngelCanelo/Insect-inspired-image-recognition-CNN. I actually run a compilation and i get a couple of errors, so i am not sure if it will work on the crazyflie. Could you look at it when you have some spare time and let me know what you think.
Regards
Rodrigo

**Comment by evoggy** [comment]
Hi!

Looking at the documentation it seems as if the Crazyflie 2.1 and the AI deck 1.1 has been used, so then it's probably safe to flash.
